generate-web/index.js

```diff
@@ -1,24 +1,159 @@
-import { saveAs } from 'file-saver'
+import "./style.css";
 
 (async () => {
   const { config } = await import('./pkg')
 
-  const elements = {
+  const DOWNLOAD_FILE_NAME = 'holo-config.json'
+
+  // Parse UI elements
+  const buttons = {
+    start: document.querySelector('#startButton'),
+    generate: document.querySelector('#generateButton'),
+    download: document.querySelector('#downloadButton'),
+    copied: document.querySelector('#copiedButton'),
+    openOlay: document.querySelector('#open-overlay'),
+    closeOlay: document.querySelector('#close-overlay')
+  }
+
+  const inputs = {
     email: document.querySelector('#email'),
-    form: document.querySelector('#form'),
-    generate: document.querySelector('#generate'),
     password: document.querySelector('#password')
   }
 
-  elements.form.addEventListener('keydown', e => {
-    if (event.key === 'Enter') { elements.generate.click() }
-  })
+  const user = {
+    email: '',
+    password: ''
+  }
```

Comment:
No need to mutate global object. Create one inside `generate` function.

Reply:
It's in global scope on purpose - `user` is used in both `generate()` and `download()`.